site2/docs/io-overview.md

```diff
@@ -1,40 +1,45 @@
 
-> #### Pulsar IO and Pulsar Functions
-> Under the hood, Pulsar IO connectors are specialized [Pulsar Functions](functions-overview.md) purpose-built to interface with external systems. The [administrative interface](io-quickstart.md) for Pulsar IO is, in fact, quite similar to that of Pulsar Functions.
+**Pulsar IO connectors** enable you to easily create, deploy, and manage connectors that interact with external systems, such as [Apache Cassandra](https://cassandra.apache.org), [Aerospike](https://www.aerospike.com), and many others.
 
-## Sources and sinks
 
-Pulsar IO connectors come in two types:
+## Concept
 
-* **Sources** feed data *into* Pulsar from other systems. Common sources include other messaging systems and "firehose"-style data pipeline APIs.
-* **Sinks** are fed data *from* Pulsar. Common sinks include other messaging systems and SQL and NoSQL databases.
+Pulsar IO connectors come in two types: **source** and **sink**.
 
-This diagram illustrates the relationship between sources, sinks, and Pulsar:
+This diagram illustrates the relationship between source, Pulsar, and sink:
 
 ![Pulsar IO diagram](assets/pulsar-io.png "Pulsar IO connectors (sources and sinks)")
 
-## Working with connectors
+### Source
 
-Pulsar IO connectors can be managed via the [`pulsar-admin`](reference-pulsar-admin.md) CLI tool, in particular the [`source`](reference-pulsar-admin.md#source) and [`sink`](reference-pulsar-admin.md#sink) commands.
+> Sources **feed data from external systems into Pulsar**. 
 
-> For a guide to managing connectors in your Pulsar installation, see the [Getting started with Pulsar IO](io-quickstart.md)
+Common sources include other messaging systems and firehose-style data pipeline APIs.
 
-The following connectors are currently available for Pulsar:
+For the complete list of Pulsar built-in source connectors, see [source connector](io-connectors.md#source-connector).
+
+### Sink
+
+> Sinks **feed data from Pulsar into external systems**. 
+
+Common sinks include other messaging systems and SQL and NoSQL databases.
+
+For the complete list of Pulsar built-in sink connectors, see [sink connector](io-connectors.md#sink-connector).
+
+## Work with connector
+
+You can manage Pulsar connectors (for example, create, update, start, stop, restart, reload, delete and perform other operations on connectors) via the [Connector CLI](reference-connector-admin.md) with [sources](reference-connector-admin.md#sources) and [sinks](reference-connector-admin.md#sinks) subcommands.
+
+## Relationship between connector and Functions
+
+Pulsar connectors are **special** [Pulsar Functions](functions-overview.md) built to interact with external systems.
+
+Sources, sinks, and Functions are components of instances, and they all run on Functions workers. When managing sources, sinks or Functions via **Pulsar admin CLI** (that is, [Connector CLI](reference-connector-admin.md) and [Functions CLI](functions-cli.md)), an instance is started on a worker.
```

Comment:
`sources, sinks or Functions` does the first letter have to be uppercase or lowercase?